src/header/header.scss

```diff
@@ -0,0 +1,201 @@
+// Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+// See LICENSE.md in the project root for license terms and full copyright notice.
+@import '../style/index';
+@import '../user-icon/index';
+@import '../icon/index';
+
+
+@mixin iui-header {
+  @include iui-reset;
+  @include iui-font-family;
+  display: flex;
+  justify-content: space-between;
+  width: 100%;
+  height: $iui-baseline * 5;
+  transition: height $iui-speed-fast ease-out;
+  @include themed {
+    background-color: t(iui-color-background-1);
+    border-bottom: $iui-xxs solid t(iui-color-background-5);
+    color: t(iui-text-color);
+  }
+
+  > .iui-left,
+  > .iui-center,
+  > .iui-right {
+    display: flex;
+    align-items: center;
+    height: 100%;
+    width: 33%;
+  }
+
+  > .iui-left {
+    &:nth-last-child(2) {
+      width: unset;
```

Comment:
Default value is `auto`

Reply:
Removed it here, but I changed it in the `iui-right`.